rcsbapi/data/schema.py

```diff
@@ -420,10 +422,14 @@ def construct_query(self, input_ids: Union[Dict[str, str], List[str]], input_typ
                 raise ValueError(
                     f"\"{return_field}\" exists, but is not a unique field, must specify further. To find valid fields with this name, run: get_unique_fields(\"{return_field}\")")
         if use_networkx:
-
-            return self.__construct_query_networkx(input_ids, input_type, return_data_list)
+            query = self.__construct_query_networkx(input_ids, input_type, return_data_list)
+        else:
+            query = self.__construct_query_rustworkx(input_ids, input_type, return_data_list)
+        validation = validate(self.client_schema, parse(query))
+        if not validation:
+            return query
         else:
-            return self.__construct_query_rustworkx(input_ids, input_type, return_data_list)
+            raise ValueError(validation)
```

Comment:
@habiba-m Can you change the name of the `validation` variable? It's currently a little confusing since a properly valid query returns an empty list, whereas an invalid query returns a list of `GraphQLError`s. Maybe change it to `validation_error_list`?